### DISCUSSION TITLE
I was concerned about the output format, so please check it.

### DISCUSSION THREAD

**Discussion by kavao** [Sanster/IOPaint#366] [General]
I have purchased. Really efficient and helpful.
Furthermore, it is very helpful if the output format is other than jpg and there is png! !
I also offered a small thank you for the cost of the coffee.

**Comment by Sanster** [comment]
Hi, thank you very much for your support. The format of the saved result image is the same as your original image. For example, if you are processing a jpg, the saved image will also be a jpg, and if you are processing a png, the result image will also be saved as png.

**Reply by kavao** [reply]
Thank you very much. I think it is high quality, but I will save it once in png for the year and then deal with lama cleaner.
